packages/create-strapi-starter/utils/fetch-github.js

```diff
@@ -4,24 +4,38 @@ const tar = require('tar');
 const fetch = require('node-fetch');
 const parseGitUrl = require('git-url-parse');
 const chalk = require('chalk');
+const stopProcess = require('./stop-process');
 
+function getShortcut(starter) {
+  let full_name;
+  // Determine if it is another organization
+  if (starter.includes('/')) {
+    const [org, project] = starter.split('/');
+    full_name = `${org}/strapi-starter-${project}`;
+  } else {
+    full_name = `strapi/strapi-starter-${starter}`;
+  }
+
+  return {
+    full_name,
+    usedShortcut: true,
+  };
+}
 /**
  * @param  {string} repo The path to repo
  */
 async function getDefaultBranch(repo) {
-  try {
-    const response = await fetch(`https://api.github.com/repos/${repo}`);
+  const response = await fetch(`https://api.github.com/repos/${repo}`);
 
-    if (!response.ok) {
-      throw Error(`Could not fetch the default branch`);
-    }
+  if (!response.ok) {
+    console.log(
+      `${chalk.red('error')} Could not fetch the default branch for: ${chalk.yellow(repo)}`
+    );
```

Comment:
if you can't fetch the default branch shouldn't you stop here or return a value ? if you just continue the code will break